cfbs.json

```
@@ -773,6 +773,42 @@
       ]
     },
     "openldap-server-not-installed": { "alias": "uninstall-openldap-server" },
+    "pkg-allowlist": {
```

I don't think you should use `pkg` in the name. Our other modules don't use abbreviations, just spell out packages. Also:

https://github.com/freebsd/pkg

https://en.wikipedia.org/wiki/.pkg

Could be: 
```suggestion
    "packages-allowlist": {
```
(Or something else).

---

ACK, done globally, including repository name.